unison-share-api/src/Unison/Server/Endpoints/DefinitionSummary.hs

```diff
@@ -41,18 +39,20 @@ import qualified Unison.Server.Backend as Backend
 import Unison.Server.Syntax (SyntaxText)
 import Unison.Server.Types
   ( APIGet,
-    ExactName (..),
     TermTag (..),
     TypeTag,
-    exactToHQ,
     mayDefaultWidth,
   )
 import qualified Unison.ShortHash as SH
 import Unison.Symbol (Symbol)
 import Unison.Util.Pretty (Width)
 
 type TermSummaryAPI =
-  "definitions" :> "terms" :> "qualified" :> Capture "fqn" (ExactName Name SH.ShortHash) :> "summary"
+  "definitions" :> "terms" :> "by-hash" :> Capture "hash" SH.ShortHash :> "summary"
+    -- Optional name to include in summary.
+    -- It's propagated through to the response as-is.
+    -- If missing, the short hash will be used instead.
```

Comment:
Just to confirm. If no `name` param is provided, it means a hash is rendered? I think this is ok, in that I should always have a name, just confirming.